bazel/antlr.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/runtime/Cpp/runtime/src/atn/ATNDeserializer.cpp b/runtime/Cpp/runtime/src/atn/ATNDeserializer.cpp
```

Comment:
any plan getting this upstream to antlr?

Reply:
You would have to talk to Kuat about that.  This is CEL.  @kyessenov

Reply:
This is coming from an internal google3 version. ANTLR broke compatibility in a patch version so it's not as easy as sending a patch.

Reply:
Can you add a comment for that, and why this is needed?

Reply:
@kyessenov if you give me the comment or submit a change to envoy-wasm I will update this PR.

Reply:
This fixes a TSan data race violation. I don't think it's a serious issue.

Reply:
It was an ASan violation according to the comment in repositories.bzl.

@lizan do you want a comment in the patch as well?